Specs/Number/Korean/NumberModel.json

```diff
@@ -2201,8 +2167,7 @@
     ]
   },
   {
-    "Input": "423 0000은 두개의 수로 인식될 것이다",
-    "IgnoreResolution": true,
+    "Input": "423 0000 은 두개의 수로 인식될 것이다",
```

Comment:
Same discussion as before. And should it really be treated as two numbers?

Reply:
That is the behavior in English ("423 0000 will be recognized as two numbers."), If there were only 3 zeros it would have been extracted as a single number

Reply:
Thanks. Once again I didn't notice the 4th "0". :-/